package.json

```diff
@@ -137,8 +137,8 @@
         "test:e2e:open": "npm-run-all -p -r start:dev cy:open",
         "test:watch": "yarn test --watch",
         "upgrade:annotations": "./build/upgrade_annotations.sh",
-        "upgrade:highlightjs": "./build/upgrade_highlightjs.sh",
-        "upgrade:pdfjs": "./build/upgrade_pdfjs.sh"
+        "upgrade:pdfjs": "./build/upgrade_pdfjs.sh",
+        "upgrade:viewers:text": "./build/upgrade_dependency.sh"
```

Comment:
Should there be any arguments here?

Reply:
I left it open for now, it runs like `yarn upgrade:viewers:text highlightjs 9.18.1`. I could add script names for `yarn upgrade:viewers:text:highlights` and the like but that seemed unnecessary

Reply:
Should we rename it to `upgrade:dependency`, in that case? It doesn't seem specific to viewers at the moment.